### PR TITLE
UNI-428 Add advanced settings

### PR DESCRIPTION
**Pull request by flysonic10**
@marionleborgne 

Refactor text to use translations config
Add advanced settings to the Create Model Dialog

### CODE REVIEW

**Comment by flysonic10** [comment]
There are a few more things to finish before this feature can be merged in:

- [ ]  Allow users to override modelRunnerParams:
   - [ ]      Aggregation Window Size
   - [x]     Aggregation Method
   - [x]     Weekly Aggregation
   - [x]     Daily Aggregation
- [x] Add lodash
- [x] Show CreateModelDialog when no aggregation is recommended
- [x] Disable radio buttons when aggregate data is unchecked
- [ ] Aggregate data recommendation text changes when it shouldn't
- [ ] Disable windowSize when aggregate data is unchecked
- [ ] Fix bug with multiple dialogs showing up
- [ ] Refactor _.get()'s to the proper formatting (obj, 'thing.thing.thing')
- [ ] Refactor AdvancedSettings into a separate component

**Comment by marionleborgne** [comment]
@lscheinkman FYI, I QA'ed the features implemented so far with @flysonic10 today and it's matching the expected behavior. 

Thanks Will, it's nicely done :+1:

**Comment by marionleborgne** [comment]
@flysonic10 one comment after a round of review with @subutai: Change the text "Learn from" to "Recognize". So that would be "Recognize daily patterns" and "Recognize daily patterns"

**Comment by marionleborgne** [comment]
I think the `lodash` dependency needs to be in the `app/package.json`

**Comment by flysonic10** [comment]
@marionleborgne This is ready to be reviewed and merged.

**Comment by marionleborgne** [comment]
There's one last linting issue:
```
numenta-apps/unicorn/app/browser/components/CreateModelDialog.jsx
  140:1  error  Trailing spaces not allowed  no-trailing-spaces
```

Make sure you have an empty line on line 140

File: unicorn/app/browser/components/AdvancedSettings.jsx [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+                </span>
+          </div>
+
+          <label style={labelStyle}>
+            {this._config.get('dialog:model:create:advanced:aggMethod:label')}
+            <IconButton
+              style={this._styles.aggregateData.tooltipIcon}
+              tooltipStyles={this._styles.aggregateData.tooltip}
+              tooltip={this._config.get('dialog:model:create:advanced:' +
+               '                         aggMethod:tooltip')}
```
Comment (marionleborgne) [inline unicorn/app/browser/components/AdvancedSettings.jsx]: This is broken because of the spaces

**Comment by marionleborgne** [comment]
Minus the small comment to address this is mergable. Thanks a lot @flysonic10 👍 💯 🚀

**Comment by flysonic10** [comment]
First merged PR. Hooray, I'm useful! 🚀

**Comment by marionleborgne** [comment]
futurama

**Pull request merged**